File: vbnet/App_Code/WebEx.Modules/bootstrap/BootstrapModule.vb

Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.Mvc
Imports WebEx.Core

<Dependency("JQuery")>
Public Class BootstrapModule
    Implements IModule

    Public Function GetViewOfType(type As String, html As HtmlHelper) As IModuleView Implements IModule.GetView
        Dim url As New UrlHelper(html.ViewContext.RequestContext)
        Select Case type
            Case "css"
                Return New ModuleViewString("<link href='{0}' rel='stylesheet'/>", url.Content("~/Content/bootstrap/bootstrap.css"))
            Case "js"
                Return New ModuleViewString("<script src='{0}'></script>", url.Content("~/Scripts/bootstrap.js"))
        End Select

        Return Nothing
    End Function
End Class

File: packages/WebEx.Bootstrap.1.0.2/content/App_Code/WebEx.Modules/bootstrap/BootstrapModule.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;
using WebEx.Core;

[Dependency("JQuery")]
public class BootstrapModule : IModule    
{    
    public IModuleView GetView(string type, HtmlHelper html)
    {
        var url = new UrlHelper(html.ViewContext.RequestContext);
        switch(type)
        {
            case "css":
                return new ModuleViewString("<link href='{0}' rel='stylesheet'/>", url.Content("~/Content/bootstrap/bootstrap.css"));
            case "js":
                return new ModuleViewString("<script src='{0}'></script>", url.Content("~/Scripts/bootstrap.js"));
        }

        return null;
    }
}